### DISCUSSION TITLE
Intervalo de data para busca.

### DISCUSSION THREAD

**Discussion by RafaelCototeIus** [gestaogovbr/Ro-dou#153] [Q&A] [answered]
Inicialmente, gostaria de agradecer a iniciativa do Governo em criar a ferramenta.

Tenho uma dúvida no parâmetro de pesquisa de data. É possível passar um período de datas, ou somente um período definido (Mês, Semana, Ano, etc), exemplo :

search:
    terms:
      - paralelpipido
    **date:** 10/10/2024 a 29/10/2024.

Obrigado!

**Comment by edulauer** [comment, marked as answer]
Olá @RafaelCototeIus, agradecemos pelo contato!

No momento o Ro-DOU permite somente os intervalos de data pré-definidos no parâmetro date (DIA, SEMANA, MES, ANO).

Mas é permitido, com base nesse intervalo, rodar a busca em uma data específica utilizando o parâmetro trigger_date ao disparar a DAG no Airflow (imagem em anexo). Desta forma, o intervalo da busca será baseado desta data de disparo.
![Captura de tela de 2024-11-10 20-47-42](https://github.com/user-attachments/assets/326fd2fe-e468-4ae2-b6ee-3e769a48d718)

Informar a data no formato AAAA-MM-DD

Caso esta forma ainda não atenda vossa necessidade, pedimos que abra uma[ issue](https://github.com/gestaogovbr/Ro-dou/issues) nesse repositório para que seja implementado em uma nova versão.
